adapter-s4j/src/main/resources/sanity-validation/e2e-nbs4j-sanity.sh

```diff
@@ -0,0 +1,147 @@
+#! /usr/local/bin/bash
```

Comment:
I'm not sure we should be adding shell scripts as assets here. Ideally, everything you need to run a complex workload can be put into named scenarios. What is the case for this one being unique enough that a different approach is needed?